Marlin/src/gcode/temp/M303.cpp

```diff
@@ -79,7 +76,7 @@ void GcodeSuite::M303() {
   const celsius_t temp = seenS ? parser.value_celsius() : default_temp;
   const bool u = parser.boolval('U');
 
-  TERN_(DWIN_LCD_PROUI, dwinStartM303(seenC, c, seenS, hid, temp));
+  TERN_(EXTENSIBLE_UI, ExtUI::onStartM303(c, hid, temp));
 
```

Comment:
`dwinStartM303()` still used, you mean to covert this over?

Reply:
I moved the call to `dwinStartM303()` to the implementation of `ExtUI::onStartM303` and it just uses the `count` and `temp` decided above.